Neos.ContentGraph.DoctrineDbalAdapter/src/Domain/Repository/ProjectionContentGraph.php

```diff
@@ -177,9 +177,38 @@ public function getAnchorPointForNodeAndOriginDimensionSpacePointAndContentStrea
 
         if (count($rows) === 1) {
             return NodeRelationAnchorPoint::fromString($rows[0]['relationanchorpoint']);
-        } else {
-            return null;
         }
+        return null;
+    }
+
+    /**
+     * @return iterable<NodeRelationAnchorPoint>
+     */
+    public function getAnchorPointsForNodeAndDimensionSpacePointsAndContentStream(
```

Comment:
what is this new method doing? I didn't find where it is used

Reply:
probably a left-over, too